subworkflows/nf-core/bam_variant_calling_freebayes/meta.yml

```diff
@@ -0,0 +1,80 @@
+name: "bam_variant_calling_freebayes"
+description: Call variants using freebayes, then sort and index
+keywords:
+  - variant
+  - sort
+  - index
+  - bam
+  - cram
+  - vcf
+modules:
+  - samtools/faidx
+  - freebayes
+  - bcftools/sort
+  - bcftools/index
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - input:
+      type: file
+      description: BAM/CRAM/SAM file
+      pattern: "*.{bam,cram,sam}"
+  - index:
+      type: file
+      description: Index BAI/CRAI/CSI file
+      pattern: "*.{bai,crai,csi}"
+  - bed:
+      type: file
+      description: Optional - Limit analysis to targets listed in this BED-format FILE.
+      pattern: "*.bed"
+  - fasta:
+      type: file
+      description: reference fasta file
+      pattern: ".{fa,fa.gz,fasta,fasta.gz}"
+  - fasta_fai:
+      type: file
+      description: reference fasta file index
+      pattern: "*.{fa,fasta}.fai"
+  - samples:
+      type: file
+      description: Optional - Limit analysis to samples listed (one per line) in the FILE.
+      pattern: "*.txt"
+  - populations:
+      type: file
+      description: Optional - Each line of FILE should list a sample and a population which it is part of.
+      pattern: "*.txt"
+  - cnv:
+      type: file
+      description: |
+        A copy number map BED file, which has either a sample-level ploidy:
+        sample_name copy_number
+        or a region-specific format:
+        seq_name start end sample_name copy_number
+      pattern: "*.bed"
+output:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. [ id:'test' ]
+  - versions:
+      type: file
+      description: File containing software versions
+      pattern: "versions.yml"
+  - vcf:
+      type: file
+      description: Sorted VCF file
+      pattern: "*.{vcf.gz}"
+  - csi:
+      type: file
+      description: Default VCF file index file
+      pattern: "*.csi"
+  - tbi:
+      type: file
+      description: Alternative VCF file index file for larger files (activated with -t parameter)
```

Comment:
If I'm not mistaken, the `csi` index is for larger files, not the `tbi`

Reply:
I borrowed from [`bcftools_index`](https://github.com/nf-core/modules/blob/master/modules/nf-core/bcftools/index/meta.yml#L39), but you are right.
> CSI (coordinate-sorted index) is created by default. The CSI format supports indexing of chromosomes up to length 2^31. TBI (tabix index) index files, which support chromosome lengths up to 2^29. When loading an index file, bcftools will try the CSI first and then the TBI.
I have updated but I `bcftools_index` needs to be updated too.